assets/dev/js/editor/regions/navigator/element.js

```diff
@@ -341,6 +342,19 @@ export default class extends Marionette.CompositeView {
 		} );
 	}
 
+	onItemPress( event ) {
+		const ENTER_KEY = 13,
+			SPACE_KEY = 32;
+
+		if ( ENTER_KEY === event.keyCode ) {
+			this.onItemClick( event );
+		}
+
+		if ( SPACE_KEY === event.keyCode ) {
+			this.onToggleListClick( event );
+		}
```

Comment:
```suggestion
		if ( ENTER_KEY === event.keyCode ) {
			this.onItemClick( event );
			return;
		}

		if ( SPACE_KEY === event.keyCode ) {
			this.onToggleListClick( event );
			return;
		}
```

should theoretically never happen. but just for readability